ballerina/types.bal

```diff
@@ -46,6 +46,70 @@ public type Options record {
     boolean enableConstraintValidation = true;  
     # If `true`, when the result is a list it will contain headers as the first row.
     boolean outputWithHeaders = false;
+    # Specifies the fail-safe options for handling errors during processing
```

Comment:
Shall we add, if enabled metadata is printed to console?

Reply:
Fixed by [289794a](https://github.com/ballerina-platform/module-ballerina-data.csv/pull/60/commits/289794af3de0c74d0b5cf00c638337e13ca07fa3)